recipes/r-ggalluvial/build.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+if [[ $target_platform =~ linux.* ]] || [[ $target_platform == win-32 ]] || [[ $target_platform == win-64 ]] || [[ $target_platform == osx-64 ]]; then
+  export DISABLE_AUTOBREW=1
+  $R CMD INSTALL --build .
+else
+  mkdir -p $PREFIX/lib/R/library/ggalluvial
+  mv * $PREFIX/lib/R/library/ggalluvial
+  if [[ $target_platform == osx-64 ]]; then
```

Comment:
Is this all needed for a functioning OSX package? Haven't seen this in any other package.

Reply:
Good call; the if statement for osx-64 shouldn't even be reached. The `build.sh` was made using the  `conda_r_skeleton_helper`; so I'll check with that team and then fix this.

Reply:
This block is emitted to recipes because it allows me to switch from building from source to binary repackaging. It is emitted by conda skeleton cran. It is neither undeliberate, a bug nor an issue.

Reply:
>   export DISABLE_AUTOBREW=1
>  $R CMD INSTALL --build .

All R packages are built from this basis.